components/HackneyDS/Breadcrumbs/index.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
+import { BreadcrumbsChevron } from '../../Icons';
 
 export default function Breadcrumbs({ values }) {
   return (
     <div className="govuk-breadcrumbs lbh-breadcrumbs">
       <ol className="govuk-breadcrumbs__list">
-        {values.map((item) => (
-          <li className="govuk-breadcrumbs__list-item" key={Math.random()}>
-            <span
-              className="govuk-breadcrumbs__link govuk-breadcrumbs__link--available"
-              onClick={(e) => {
-                e.preventDefault();
-                item.onClick();
-              }}
-            >
-              {item.text}
-            </span>
-          </li>
-        ))}
+        {values.map((item, index) => {
+            const isLastItem = index + 1 === values.length;
+            return (
+              <li className="govuk-breadcrumbs__list-item" key={Math.random()}>
```

Comment:
Better to replace `Math.random()` with `item.text`. Text is supposed to be unique in breadcrumbs